debugger/src/main/scala/org.apache.daffodil.debugger.dap/Parse.scala

```diff
@@ -399,8 +399,10 @@ object Parse {
     // arguments: Launch config
     def parseRootName(arguments: JsonObject) =
       Right(
-        Option(arguments.getAsJsonObject("schema").getAsJsonPrimitive("rootName"))
-          .map(_.getAsString())
+        arguments.getAsJsonObject("schema").get("rootName") match {
```

Comment:
This change was needed for both `parseRootName` and `parseRootNamespace`. Without it, passing `null` from the TS to the Scala would cause an error:

```bash
class com.google.gson.JsonNull cannot be cast to class com.google.gson.JsonPrimitive
```